tanjun/__init__.py

```diff
@@ -112,6 +112,8 @@ async def main() -> None:
     "ClientCallbackNames",
     # checks.py
     "checks",
+    "all_checks",
```

Comment:
Do we want to add with_any_checks and with_all_checks decorators and expose them on this level instead?